pandas/tests/groupby/test_groupby.py

```diff
@@ -961,10 +983,20 @@ def test_groupby_with_hier_columns():
     tm.assert_index_equal(result.columns, df.columns[:-1])
 
 
-def test_grouping_ndarray(df):
+def test_grouping_ndarray(df, using_infer_string):
     grouped = df.groupby(df["A"].values)
```

Comment:
Maybe convert the columns to object dtype here, so the rest of the test can remain the same? (as you did in some other tests as well) 
Because it seems this test is covering the case of passing an array-like (series) as grouper, not specifically testing the sum operation